src/app/shared/log-in/log-in.component.ts

```diff
@@ -1,9 +1,9 @@
-import { filter, map, takeWhile } from 'rxjs/operators';
-import { Component, OnDestroy, OnInit } from '@angular/core';
+import {filter, map, takeWhile } from 'rxjs/operators';
```

Comment:
In this file some imports have spaces before/after curly braces but some imports don't have. in general spaces there should be

Reply:
Thanks! I updated typescript settings in intellij to add the spaces & fixed these